ddtrace/contrib/django/conf.py

```diff
@@ -22,13 +22,14 @@
 
 # List of available settings with their defaults
 DEFAULTS = {
-    'TRACER': 'ddtrace.tracer',
-    'DEFAULT_SERVICE': 'django',
-    'ENABLED': True,
-    'AUTO_INSTRUMENT': True,
     'AGENT_HOSTNAME': 'localhost',
     'AGENT_PORT': 8126,
+    'AUTO_INSTRUMENT': True,
+    'DEFAULT_DATABASE_PREFIX': '',
```

Comment:
Cool! I think we should just add `DEFAULT_DATABASE_PREFIX` to [the docs](https://github.com/jairhenrique/dd-trace-py/blob/107e623b4d6f68625ef5c99c3c9cda86fd07305e/ddtrace/contrib/django/__init__.py#L40) so that people know how to use that setting.